Duality.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.40629.0
+# Visual Studio 15
+VisualStudioVersion = 15.0.27428.1
```

Comment:
Technically not a clean PR with this change, as updating the VS version of the solution could backfire and wasn't part of the issue - but I'll take this as it make sense to update the solution version by now. Would prefer a more selective PR next time though 🙂